Development/nmos-cpp-node/node_implementation.cpp

```diff
@@ -343,13 +344,16 @@ void node_implementation_init(nmos::node_model& model, slog::base_gate& gate)
             nmos::resource flow;
             if (impl::ports::video == port)
             {
-                flow = nmos::make_raw_video_flow(
+                flow = nmos::make_coded_video_flow(
                     flow_id, source_id, device_id,
                     frame_rate,
                     frame_width, frame_height, interlace_mode,
-                    colorspace, transfer_characteristic, sampling, bit_depth,
+                    colorspace, transfer_characteristic, nmos::media_types::video_H264,
                     model.settings
                 );
+                flow.data[nmos::fields::components] = nmos::make_components(sampling, frame_width, frame_height, bit_depth);
+                // Baseline Profile, Level 1
+                flow.data[nmos::fields::profile_level_id] = value(0x42000A);
```

Comment:
I can imagine adding an overload of `nmos::make_coded_video_flow` with a `sampling` argument to avoid this little dance with `nmos::fields::components`... it would be useful for JPEG XS for example too. On the other hand, for `nmos::fields::profile_level_id` that would perhaps need a "video/H264" specific function like `nmos::make_video_H264_flow` **or** to wait for an AMWA decision for the NMOS Parameter Registers and future BCP-006-xxx specs that a common reusable `profile_level` (or separate `profile` and `level`) parameter was the way forward.